docs/screenplay_elements/Web/Actions/click.md

```diff
@@ -34,15 +35,33 @@ public async performAs(actor: Actor): Promise<void>;
 #### on
 
 ```typescript
-public static on(selector: Selector, options?: SelectorOptions): Click;
+public static on(selector: Selector, options?: SelectorOptions & ClickOptions): Click;
 ```
 
 - **Description:** Creates a new instance of the `Click` class specifically for clicking on an element specified by a selector.
 - **Parameters:**
-  - `selector` - The Selector.
-  - `options` (optional) - Advanced selector lookup options.
+  - `selector` - The Selector (can be a string or a Playwright Locator).
```

Comment:
If we made that remark everywhere then go for it, else I wouldnt mention it just here in that single action